pricelist_cache_rest/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Provides an endpoint to get product prices for a given customer
```

Comment:
```suggestion
Provides an endpoint to expose product prices cached by `pricelist_cache`.
Endpoint authorization is based on `auth_api_key`.
```
then pls add a `USAGE.rst` module to explain how to use like:

```rst
The endpoint requires the partner ID of the customer prices for which you want to retrieve prices.
It should be called like this::

    curl -X GET "http://your.odoo/pricelist/$partner_id" -H "accept: /" -H "API-KEY: XYZ"

By default you'll get a list of dict like this::

    [
        {"id": $product_id, "price": $price},
        {"id": $product_id, "price": $price},
        {"id": $product_id, "price": $price},
    ]

You can modify the `ir.exports` configuration named "Pricelist Cache Parser" to change this schema.
```
Then add a `CONFIGURATION.rst` to describe how to configure api keys and set them on the company.